ClientPlugin/GUI/Constructor.cs

```diff
@@ -0,0 +1,85 @@
+﻿using ClientPlugin.GUI.Elements;
+using Sandbox;
+using Sandbox.Graphics.GUI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VRageMath;
+using VRage.Render11;
+using VRage.Utils;
+using DirectShowLib.DES;
+
+namespace ClientPlugin.GUI
```

Comment:
The `GUI` folder is not part of the project and appears to have unimplemented/unfinished code in it. Leftover files from an earlier stage of implementation?

Reply:
Deleted in https://github.com/sepluginloader/PluginTemplate/pull/22/commits/3013ee4ce962ed96d1c716fb618fc5cd76488ef0